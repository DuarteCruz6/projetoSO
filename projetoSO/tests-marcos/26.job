WRITE [(u4,83)(o2,49)(g6,90)]
SHOW
WRITE [(b5,8)(e6,7)(p8,15)(z2,33)(n5,49)]
WRITE [(r9,60)(g9,64)]
READ [r9]
DELETE [e6,u4,r9]
READ [p8]
READ [n5]
DELETE [p8,g6,b5,o2,z2]
DELETE [g9]
WRITE [(t2,50)]
WRITE [(r3,59)(z9,22)]
DELETE [r3,z9,t2,n5]
WRITE [(f9,23)(m4,49)(s6,13)(l9,6)]
WRITE [(z3,72)]
DELETE [s6,f9]
WRITE [(s8,30)(o5,33)]
WRITE [(o6,29)(v1,9)(d2,20)(a7,4)]
WRITE [(r2,74)(i9,68)(r7,63)(n3,77)(g2,24)]
WRITE [(p9,79)(i8,62)(i0,87)(u1,32)(f6,71)]
WRITE [(f8,79)(p6,6)(m6,63)(b6,38)]
WAIT 1000
WRITE [(o2,61)(z7,63)(u2,35)]
READ [v1]
WRITE [(c0,30)(a8,89)(w0,25)]
DELETE [a8,v1,c0,b6,d2]
WRITE [(u0,63)(t9,46)(n5,79)(y4,56)(w6,55)]
WRITE [(d8,75)]
WRITE [(l5,82)(b8,10)(x3,14)]
WRITE [(z9,67)(x0,17)(w1,62)(d6,15)(g4,89)]
WRITE [(z4,93)(e5,46)(q0,78)(y5,86)(k0,88)]
WRITE [(r0,81)(h9,97)]
READ [n3]
WRITE [(q8,85)]
WRITE [(b4,31)(e7,15)(q3,34)]
READ [r7]
WRITE [(o3,8)(x8,3)]
DELETE [o6,n3,k0,o5,g2]
WRITE [(t1,20)(y3,24)(p1,75)(o4,95)]
DELETE [r7,i0]
DELETE [z9,z4,m4]
WRITE [(s9,27)(j7,23)(w7,11)(m3,75)(x4,6)]
READ [l5]
WRITE [(k5,78)(w8,8)(p5,31)(w3,45)(w5,60)]
WRITE [(d3,15)(f2,18)(o7,41)(n9,74)]
WRITE [(m1,50)(g8,19)(t8,87)(m2,24)(f9,94)]
WAIT 3000
READ [n5]
READ [u0]
WAIT 1000
WRITE [(l2,22)(b6,93)(i3,38)(h1,34)(w4,47)]
WRITE [(t5,50)(u3,13)(g9,61)(l8,77)(l7,92)]
READ [u0]
WRITE [(c4,36)(e9,95)(x6,35)(s6,57)]
WRITE [(n3,53)(e1,46)(c5,60)(i6,74)]
WRITE [(k8,89)(x1,4)(j5,61)(h7,46)(p2,13)]
READ [l2]
READ [f8]
READ [w0]
DELETE [e1,p2,b6]
WRITE [(e3,38)(x9,43)(e6,22)(u8,67)]
WRITE [(h5,30)(j6,62)]
READ [h7]
BACKUP
WRITE [(j0,7)(i5,71)(w9,47)]
WRITE [(g2,37)(q5,46)(b0,23)(e4,72)]
BACKUP
WRITE [(r1,15)(k2,91)(m7,93)]
WRITE [(c0,46)(l3,25)(o6,81)(y9,69)(y6,13)]
WRITE [(m0,29)(e2,40)(b6,3)]
WRITE [(r5,53)(n4,67)(j8,34)(l1,77)(u4,23)]
DELETE [c0]
READ [y6]
DELETE [x9,b8,w7,g9,x4]
BACKUP
WRITE [(v0,3)(p2,47)(n0,34)(s1,74)(c9,73)]
DELETE [t8,x6,u2,g8]
BACKUP
WRITE [(d4,18)(v6,70)]
DELETE [l9,c5]
READ [w5]
READ [m1]
DELETE [l8,c9,w0]
READ [n0]
READ [l3]
