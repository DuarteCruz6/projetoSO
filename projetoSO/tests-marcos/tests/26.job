WRITE [(n2,95)]
READ [n2]
DELETE [n2]
WRITE [(j0,27)(p8,41)(p5,0)]
READ [p8]
READ [j0]
WRITE [(m4,32)]
WAIT 2000
WRITE [(w1,78)(x3,69)(v7,93)(m5,56)]
SHOW
WRITE [(s7,85)(r2,77)(y0,94)(w9,65)(c3,96)]
DELETE [r2,s7,c3]
READ [v7]
READ [m4]
READ [p8]
BACKUP
READ [p5]
READ [v7]
WRITE [(n6,84)(p3,12)(t5,76)(p1,89)(y6,72)]
WRITE [(v4,91)(t3,60)(w6,42)(t7,19)]
READ [m5]
DELETE [v7,w1,y0]
DELETE [m4,p8]
READ [j0]
DELETE [t7,j0,n6,t5,w9]
DELETE [p1]
READ [w6]
WRITE [(v2,63)]
DELETE [t3]
READ [m5]
WRITE [(u4,68)(b4,83)(n8,87)(z1,66)]
WRITE [(z6,29)(a1,29)]
WRITE [(f7,20)(t3,59)(i6,7)(w4,82)]
DELETE [z1,a1,f7]
WRITE [(b5,11)(j8,37)(j4,4)]
WRITE [(g7,77)(n9,43)(x9,37)(h5,33)(f6,18)]
WRITE [(l7,69)(a6,1)(g1,54)(i3,22)(l2,23)]
WAIT 2000
WRITE [(b3,28)(k0,8)]
DELETE [u4]
READ [n8]
READ [p3]
READ [g7]
WRITE [(m0,95)]
READ [j4]
