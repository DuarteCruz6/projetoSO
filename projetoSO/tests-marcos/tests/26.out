[(n2,95)]
[(p8,41)]
[(j0,27)]
(j0, 27)
(m5, 56)
(m4, 32)
(p5, 0)
(p8, 41)
(v7, 93)
(w1, 78)
(x3, 69)
[(v7,93)]
[(m4,32)]
[(p8,41)]
[(p5,0)]
[(v7,93)]
[(m5,56)]
[(j0,27)]
[(w6,42)]
[(m5,56)]
[(n8,87)]
[(p3,12)]
[(g7,77)]
[(j4,4)]
